TLM/TMPE.API/TMPE.API.csproj

```diff
@@ -185,6 +180,7 @@
     <Analyzer Include="..\packages\StyleCop.Analyzers.1.1.118\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
     <Analyzer Include="..\packages\StyleCop.Analyzers.1.1.118\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
?

Reply:
```suggestion
```

Reply:
We could remove it. When it comes to generated files, I tend to just let Visual Studio do its thing unless I have a specific concern.

Reply:
remove imo

Reply:
Done